src/ClaudeEngine.php

```
@@ -0,0 +1,56 @@
+{
+    public function __construct(
+        private ?string $model = null
+    ) {
+    }
+
+    public function translate(string $text, string $systemPrompt): string
+    {
+        $output = $this->callClaude($text, $systemPrompt);
+        
+        if ($output === false || trim($output) === '') {
+            throw new \RuntimeException('Failed to get response from Claude CLI');
```

PHPMD (2.15.0)

Missing class import via use statement (line '19', column '23'). (Clean Code Rules)

(MissingImport)